src/Dapr.Workflow/WorkflowServiceCollectionExtensions.cs

```diff
@@ -14,16 +14,21 @@
 namespace Dapr.Workflow
 {
     using System;
+    using Grpc.Net.Client;
     using Microsoft.DurableTask.Client;
     using Microsoft.DurableTask.Worker;
     using Microsoft.Extensions.DependencyInjection;
     using Microsoft.Extensions.DependencyInjection.Extensions;
+    using System.Net.Http;
+    using Dapr;
 
     /// <summary>
     /// Contains extension methods for using Dapr Workflow with dependency injection.
     /// </summary>
     public static class WorkflowServiceCollectionExtensions
     {
+        private static string daprApiToken = "";
```

Comment:
Why did we add this is a private static field? `DaprDefaults` does the lazy load for us so we have no need to create a static variable here.